File: ClientApp/src/app/models/extensionStatus.ts
export class ExtensionStatus {
  constructor(
    public ID: string,
    public Status?: string,
    public SubStatus?: string,
    public LastActivity?: string,
    public LastActivityIdle?: string,
    public ITime?: number,
    public ATime?: number,
    public CTime?: number,
    public TotalCalls?: number,
    public TotalICalls?: number,
    public TotalOCalls?: number,
    public LoggedInTime?: number,
    public IdleTimeList?: number[],
    public StatusList?: string[],
    public SubStatusList?: string[],
    public DurationList?: number[],
    public LogoutTimesList?: Date[],
  ) { }
}

File: Models/ExtensionStatus.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace PBXDashboard_Dev.Models
{
    public class ExtensionStatus
    {
        public string ID { get; set; }
        public string Status { get; set; }
        public string LastActivity { get; set; }
        public string LastActivityIdle { get; set; }
        public double ITime { get; set; }
        public double ATime { get; set; }
        public double CTime { get; set; }
        public int TotalCalls { get; set; }
        public int TotalICalls { get; set; }
        public int TotalOCalls { get; set; }
        public double LoggedInTime { get; set; }
        public string SubStatus { get; set; }
        [NotMapped]
        public List<string> StatusList { get; set; }
        [NotMapped]
        public List<string> SubStatusList { get; set; }
        [NotMapped]
        public List<double> DurationList { get; set; }
        [NotMapped]
        public List<DateTime> LogoutTimesList { get; set; }
        [NotMapped]
        public List<double> IdleTimeList { get; set; }
    }
}